medicai/transforms/base.py

```
@@ -321,6 +341,287 @@ def record_random_transform(
+            return bundle
+
+        should_apply = self.sample_should_apply()
+        permutation = self._sample_permutation_graph()
+        num_to_apply = self._sample_num_choices()
+
+        def apply_step(current_outputs, step: int):
+            selected_index = permutation[step]
+
+            def make_branch(index: int, current_outputs=current_outputs):
+                def branch():
+                    local_data = {key: value for key, value in zip(data_keys, current_outputs)}
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)